x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/scalar/math/Tanh.java

```diff
@@ -29,15 +29,15 @@ public class Tanh extends AbstractTrigonometricFunction {
 
     @FunctionInfo(
         returnType = "double",
-        description = "Returns the {wikipedia}/Hyperbolic_functions[Tangent] hyperbolic function of an angle.",
+        description = "Returns the {wikipedia}/Hyperbolic_functions[Tangent] hyperbolic function of a number.",
```

Comment:
nit: for consistency, we could have
```suggestion
        description = "Returns the {wikipedia}/Hyperbolic_functions[hyperbolic tangent] of a number.",
```